src/jit/emitxarch.cpp

```diff
@@ -8712,12 +8719,23 @@ void emitter::emitDispIns(
             break;
 
         case IF_RWR_RRD_RRD:
+        {
             assert(IsAVXInstruction(ins));
             assert(IsThreeOperandAVXInstruction(ins));
+            regNumber reg2 = id->idReg2();
+            regNumber reg3 = id->idReg3();
+            if (ins == INS_bextr || ins == INS_bzhi)
```

Comment:
It would be good to add a comment explaining why the regs are reversed for this case.